codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectEncoder.java

```diff
@@ -47,10 +47,10 @@
  * implement all abstract methods properly.
  */
 public abstract class HttpObjectEncoder<H extends HttpMessage> extends MessageToMessageEncoder<Object> {
-    static final byte[] CRLF = { CR, LF };
-    private static final byte[] ZERO_CRLF = { '0', CR, LF };
+    protected static final int CRLF_SHORT = (CR << 8) | LF;
```

Comment:
I wonder if we should avoid exposing this outside the package. Once we introduce publicly visible API elements it is tough to remove them later. @normanmaurer - wdyt?